test_analyzer_streamline_single.py

```diff
@@ -34,19 +35,47 @@
 	yhigh = float(raw_input('Y high: '))
 
```

Comment:
I tried running your code yesterday, and if your code expects certain input at runtime, it's really helpful to provide the alternatives along with the prompt. Like `Toggle EEP? (Y/N) `